spec/unit/util/storage_spec.rb

```diff
@@ -173,6 +173,7 @@ def write_state_file(contents)
       @state_file = tmpfile('storage_test')
       @saved_statefile = Puppet[:statefile]
       Puppet[:statefile] = @state_file
+      @A_SMALL_AMOUNT_OF_TIME = 0.001 #Seconds
```

Comment:
ah one nit, some older spec tests use instance variables in the `before` block, because `let` came later. So that's why the `@saved_statefile` is there. But for a constant like above, I'd just make that a constant in the outer describe block:

```ruby
describe '...' do
  A_SMALL_AMOUNT_OF_TIME = 0.001
end
```

Reply:
OK thanks.  let wasn't working and I (guess I still don't) know why, but this worked.